### PR TITLE
feat(RHTAPREL-806): update operator toolkit refrence

### PR DESCRIPTION
**Pull request by happybhati**
This commit updates the `operator-toolkit` refrence
 as a result of migration to `konflux-ci` from `redhat-appstudio`
 more details parent EPIC:
 https://issues.redhat.com/browse/RHTAPREL-800

### CODE REVIEW

**Comment by happybhati** [comment]
`Do Not Merge/Hold` preparation PR, Can be merged once repo is migrated to `konflux-ci`!

**Review by davidmogar — APPROVED** [review]

**Pull request merged**
